nextjs-app/components/pages/program-rules/Schedule/DetailCard.tsx

```diff
@@ -48,7 +54,7 @@ const DetailCard: FC<IScheduleDetail> = ({ phase, type, timeline, event }) => {
   const renderEvent = () => {
```

Comment:
```suggestion
  const renderEvent = () => {
    if (!Array.isArray(event)) {
      return <DetailContent phase={phase} event={event} isSingleMessage />;
    }
    
    return (
      <div className="w-full flex flex-wrap lg:flex-nowrap gap-5 lg:gap-9 justify-center md:justify-start">
        {event.map((e, index) => (
          <DetailContent
            key={`${e.title}-${index}`}
            event={e}
            phase={phase}
            isDivider={index < event.length - 1}
          />
        ))}
      </div>
    );
  };
```

Will this be more readable ?

Reply:
You're right!
It's been updated.